feature-libs/product-configurator/rulebased/components/price/configurator-price.component.spec.ts

```diff
@@ -89,45 +89,90 @@ describe('ConfiguratorPriceComponent', () => {
       CommonConfiguratorTestUtilsService.expectElementPresent(
         expect,
         htmlElem,
-        '.cx-price'
+        '.cx-greyed-out'
       );
       CommonConfiguratorTestUtilsService.expectElementNotPresent(
         expect,
         htmlElem,
         '.cx-quantity-price'
       );
 
-      expect(component.price).toEqual('+ $10');
+      expect(component.price).toEqual('+$10');
     });
 
-    it('should be defined: value price greater than zero and value is selected', () => {
-      component.formula = createTestData(0, 10, 10, true);
+    it('should be defined: negative value price is shown correctly', () => {
+      component.formula = createTestData(0, -10, 0, false, true);
+      component.formula = {
```

Comment:
You try to define formula twice. Plz, remove the second one

Reply:
done